txpipe/diagnostics.py

```diff
@@ -1,7 +1,7 @@
 from .base_stage import PipelineStage
 from .data_types import Directory, HDFFile, PNGFile, TomographyCatalog
 from .utils.stats import ParallelStatsCalculator
-from .utils.metacal import apply_response
+from .utils.metacal import calculate_selection_bias, calculate_multiplicative_bias, apply_metacal_response
```

Comment:
Thanks for putting these in a wider library, they'll be v useful for other stuff elsewhere I think.